server/app.js

```diff
@@ -102,6 +87,7 @@ const createApp = ({
   app.use(
     helmet({
       contentSecurityPolicy: false,
+      referrerPolicy: { policy: ['no-referrer', 'same-origin'] },
```

Comment:
default to no-referrer if same origin is not specfied.

same-origin should ensure allowlisted sites do not receive referral information